bin/hashbangctl

```diff
@@ -129,7 +129,10 @@ while True:
     elif key == 's':
         shell = raw_input("Enter new shell, eg /bin/bash:")
 
-        if shell not in ['/bin/bash','/bin/zsh','/bin/ksh','/bin/fish']:
+    with open('/etc/shells') as shells_list:
```

Comment:
The indentation is still wrong: this whole code block you are adding should be under the `elif key == 's'` conditional.